src/assets/interactables/duck/gltf/DuckyMesh.gltf.meta

```diff
@@ -0,0 +1 @@
+{"asset":{"generator":"FBX2glTF","version":"2.0"},"scene":0,"buffers":[{"byteLength":18600,"uri":"buffer.bin"}],"bufferViews":[{"buffer":0,"byteLength":3336,"byteOffset":0,"target":34963},{"buffer":0,"byteLength":5724,"byteOffset":3336,"target":34962},{"buffer":0,"byteLength":5724,"byteOffset":9060,"target":34962},{"buffer":0,"byteLength":3816,"byteOffset":14784,"target":34962}],"scenes":[{"name":"Root Scene","nodes":[0]}],"accessors":[{"componentType":5123,"type":"SCALAR","count":1668,"bufferView":0,"byteOffset":0},{"componentType":5126,"type":"VEC3","count":477,"bufferView":1,"byteOffset":0,"min":[-0.0587991699576378,0.0129461474716663,-0.0740185976028442],"max":[0.0618169121444225,0.173104390501976,0.104356855154037]},{"componentType":5126,"type":"VEC3","count":477,"bufferView":2,"byteOffset":0},{"componentType":5126,"type":"VEC2","count":477,"bufferView":3,"byteOffset":0}],"images":[{"name":"DuckyMesh.fbm/Ducky.jpg","uri":"DuckyMesh.fbm/Ducky.jpg"}],"samplers":[{}],"textures":[{"name":"file4","sampler":0,"source":0}],"materials":[{"name":"lambert2","alphaMode":"OPAQUE","extras":{"fromFBX":{"shadingModel":"Lambert","isTruePBR":false}},"pbrMetallicRoughness":{"baseColorTexture":{"index":0,"texCoord":0},"baseColorFactor":[0.800000011920929,0.800000011920929,0.800000011920929,1],"metallicFactor":0.200000002980232,"roughnessFactor":0.800000011920929}}],"meshes":[{"name":"Ducky","primitives":[{"material":0,"mode":4,"attributes":{"NORMAL":2,"POSITION":1,"TEXCOORD_0":3},"indices":0}]}],"nodes":[{"name":"RootNode","translation":[0,0,0],"rotation":[0,0,0,1],"scale":[1,1,1],"children":[1]},{"name":"Ducky","translation":[0,0,0],"rotation":[0,0,0,1],"scale":[1,1,1],"mesh":0}]}
```

Comment:
Whats this .meta file?

Reply:
I have no idea. It got generated when I exported the .gltf model. Are those not just part of the gltf? I can delete it.